src/static/js/modules/nav-primary.js

```diff
@@ -28,16 +37,26 @@ function init() {
     var $this = $( this );
     var $thisSubNav = $this.siblings( '.js-sub-nav' );
     var $otherSubNavs = $subNavs.not( $thisSubNav );
+    var $firstLink = $thisSubNav.find( 'a' )[0];
```

Comment:
No, it would still work because it would be using the native `focus` method. I was just wondering if would should prefix elements with `$` when they aren't jQuery Elements.